docs/Tutorial_DiscordMERLIN.ipynb

```diff
@@ -0,0 +1,2005 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #93.        r = r_init
</pre><p>And then here, you can just do <code>r_copy = r</code>  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/505/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>

Reply:
<p>Got it. Will be fixed in next set of commits.</p>

Reply:
@seanlaw 
Just to confirm.... did you mean to use the exact name `r_copy` inside while-loop?

Reply:
Yes. Either `r_copy` or maybe even `r_updated` (since it is updated in each iteration)?

Reply:
<p>I was thinking about <code>min_dist = r</code>... but I think <code>r_updated</code> is better as we already have two value <code>max_dist</code> and <code>config.STUMPY_MIN_DISCORD_DISTANCE</code>  that are fixed and not being updated... so, I will use <code>r_updated</code></p>